i18n/ko.json

```
@@ -1,1962 +1,1962 @@
+    "server_settings": "서버 설정",
+    "server_settings_description": "서버 관련 설정 관리",
+    "server_welcome_message": "환영 메시지",
+    "server_welcome_message_description": "로그인 페이지에 표시되는 메시지입니다.",
+    "sidecar_job": "사이드카 메타데이터",
+    "sidecar_job_description": "파일 시스템에서 사이드카 메타데이터를 검색하거나 동기화합니다.",
+    "slideshow_duration_description": "각 이미지를 표시할 시간 (초)",
+    "smart_search_job_description": "기계 학습을 통해 스마트 검색을 위한 데이터 처리",
+    "storage_template_date_time_description": "항목의 생성 시간을 날짜/시간 정보로 사용합니다.",
+    "storage_template_date_time_sample": "예시 시간 {date}",
+    "storage_template_enable_description": "저장 템플릿 엔진 활성화",
+    "storage_template_hash_verification_enabled": "해시 검증 활성화됨",
+    "storage_template_hash_verification_enabled_description": "해시 검증을 활성화합니다. 영향에 대해 확실하지 않다면 비활성화하지 마세요.",
+    "storage_template_migration": "저장 템플릿 마이그레이션",
+    "storage_template_migration_description": "현재 <link>{template}</link> 템플릿을 기존 업로드 항목에 적용합니다.",
+    "storage_template_migration_info": "이 템플릿은 확장자를 모두 소문자로 변환합니다. 변경 사항은 새 항목에만 적용되며, 기존 항목에 소급 적용하려면 <link>{job}</link> 작업을 실행하세요.",
+    "storage_template_migration_job": "저장 템플릿 마이그레이션 작업",
+    "storage_template_more_details": "이 기능에 대한 자세한 내용은 <template-link>저장 템플릿</template-link>과 <implications-link>영향</implications-link> 문서를 참조하세요.",
+    "storage_template_onboarding_description": "이 기능을 활성화하면 사용자 정의 템플릿에 따라 파일이 자동 정리됩니다. 안정성 문제로 인해 기본적으로 꺼져 있습니다. 자세한 내용은 <link>문서</link>를 참조하세요.",
+    "storage_template_path_length": "예상 경로 길이 제한: <b>{length, number}</b>/{limit, number}",
+    "storage_template_settings": "저장 템플릿",
+    "storage_template_settings_description": "업로드 항목의 폴더 구조 및 파일 이름 설정 관리",
+    "storage_template_user_label": "<code>{label}</code>은 사용자의 스토리지 라벨입니다.",
+    "system_settings": "시스템 설정",
+    "tag_cleanup_job": "태그 정리",
+    "template_email_available_tags": "템플릿에서 다음 변수를 사용할 수 있습니다: {tags}",
+    "template_email_if_empty": "템플릿이 비어 있으면 기본 이메일이 사용됩니다.",
+    "template_email_invite_album": "앨범 초대 이메일 템플릿",
+    "template_email_preview": "미리보기",
+    "template_email_settings": "이메일 템플릿",
+    "template_email_settings_description": "사용자 정의 이메일 알림 템플릿 관리",
+    "template_email_update_album": "앨범 업데이트 이메일 템플릿",
+    "template_email_welcome": "환영 이메일 템플릿",
+    "template_settings": "알림 템플릿",
+    "template_settings_description": "사용자 정의 알림 템플릿 관리",
+    "theme_custom_css_settings": "사용자 정의 CSS",
+    "theme_custom_css_settings_description": "CSS를 통해 Immich의 디자인을 커스터마이즈할 수 있습니다.",
+    "theme_settings": "테마 설정",
+    "theme_settings_description": "Immich 웹 인터페이스의 사용자 정의 관리",
+    "these_files_matched_by_checksum": "이 파일들은 체크섬 기준으로 일치합니다.",
+    "thumbnail_generation_job": "썸네일 생성",
+    "thumbnail_generation_job_description": "각 항목 및 인물에 대해 크고 작은 썸네일, 흐릿한 썸네일을 생성합니다.",
+    "transcoding_acceleration_api": "가속 API",
+    "transcoding_acceleration_api_description": "트랜스코딩을 가속화하기 위해 장치와 상호작용하는 API입니다. 이 설정은 '최선의 노력' 방식으로, 실패 시 소프트웨어 방식으로 자동 전환됩니다. VP9은 하드웨어에 따라 지원되지 않을 수 있습니다.",
```

```suggestion
    "transcoding_acceleration_api_description": "트랜스코딩을 가속화하기 위해 장치와 상호작용하는 API입니다. 이 설정은 최선의 노력(best effort) 방식으로, 실패 시 소프트웨어 방식으로 자동 전환됩니다. VP9은 하드웨어에 따라 지원되지 않을 수 있습니다.",
```
The phrase “최선의 노력” isn’t commonly used in translations, so including the English term (best effort) alongside it makes the sentence sound more natural.

---

http://www.ktword.co.kr/test/view/view.php?no=1392

한국정보통신기술협회 등록 단어입니다. (TTA)